instance/models/mixins/openedx_config.py

```diff
@@ -56,7 +56,8 @@ def _get_configuration_variables(self):
                     "openedx.core.djangoapps.heartbeat",
                 ],
                 "HEARTBEAT_EXTENDED_CHECKS": [
-                    "lms.lib.comment_client.utils.check_forum_heartbeat",
```

Comment:
@swalladge, since the path was changed only after the Ironwood release, removing this will break all the instances running Ironwood or an older version. The way we usually deal with it is to add a conditional checking for all the releases which use this and set the old value if it is true or if not, set the new value.

Reply:
Please see the latest changes - I added conditional checking (untested). It's a bit messy because it appears the path changed between ironwood.1 and ironwood.2. Is this the kind of conditional checking you'd like to see?

I guess a neat alternative would be to disable forum heartbeat checking by default, and we can enable it manually for instances where we need it?